superbench/benchmarks/model_benchmarks/model_base.py

```diff
@@ -131,6 +131,13 @@ def add_parser_arguments(self):
             help='Disable GPU training.',
         )
 
+        self._parser.add_argument(
+            '--pin_memory',
+            action='store_true',
+            default=False,
```

Comment:
should we enable `pin_memory` by default?

Reply:
We can discuss whether we need to change it in other PR.